src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,21 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
+        excavator.setModel("CAT");
+        Bulldozer bulldozer = new Bulldozer();
+        bulldozer.setModel("John Deere");
+        Truck truck = new Truck();
+        truck.setModel("Ford");
```

Comment:
Look at the [checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop). Let's use abstract reference instead of specific one